pkgs/applications/graphics/glimpse/plugins/default.nix

```diff
@@ -0,0 +1,199 @@
+      hardeningDisable = [ "format" ];
+      meta = with stdenv.lib; {
+        description = "The GIMP Animation Package";
+        homepage = "https://www.gimp.org";
+        # The main code is given in GPLv3, but it has ffmpeg in it, and I think ffmpeg license
+        # falls inside "free".
+        license = with licenses; [ gpl3 free ];
+      };
+    };
+
+    fourier = pluginDerivation rec {
+      # menu:
+      # Filters/Generic/FFT Forward
+      # Filters/Generic/FFT Inverse
+      name = "fourier-0.4.3";
+      buildInputs = with pkgs; [ fftw ];
+
+      src = fetchurl {
+        url = "https://www.lprp.fr/files/old-web/soft/gimp/${name}.tar.gz";
+        sha256 = "0mf7f8vaqs2madx832x3kcxw3hv3w3wampvzvaps1mkf2kvrjbsn";
+      };
+
+      installPhase = "installPlugins fourier";
+
+      meta = with stdenv.lib; {
+        description = "GIMP plug-in to do the fourier transform";
+        homepage = "https://people.via.ecp.fr/~remi/soft/gimp/gimp_plugin_en.php3#fourier";
+        license = with licenses; [ gpl3Plus ];
+      };
+    };
+
+    resynthesizer = pluginDerivation rec {
+      # menu:
+      # Edit/Fill with pattern seamless...
+      # Filters/Enhance/Heal selection...
+      # Filters/Enhance/Heal transparency...
+      # Filters/Enhance/Sharpen by synthesis...
+      # Filters/Enhance/Uncrop...
+      # Filters/Map/Style...
+      # Filters/Render/Texture...
+      pname = "resynthesizer";
+      version = "2.0.3";
+      buildInputs = with pkgs; [ fftw ];
+      nativeBuildInputs = with pkgs; [ autoreconfHook ];
+      configureFlags = [ "--libdir=${placeholder "out"}/lib/glimpse/2.0" ];
+      makeFlags = [ "GIMP_LIBDIR=${placeholder "out"}/lib/glimpse/2.0" ];
+      src = fetchFromGitHub {
+        owner = "bootchk";
+        repo = "resynthesizer";
+        rev = "v${version}";
+        sha256 = "1jwc8bhhm21xhrgw56nzbma6fwg59gc8anlmyns7jdiw83y0zx3j";
+      };
+    };
+
+    waveletSharpen = pluginDerivation {
+      # menu:
+      # Filters/Enhance/Wavelet sharpen
+      name = "wavelet-sharpen-0.1.2";
+      NIX_LDFLAGS = [ "-lm" ];
+      src = fetchurl {
+        url = "https://github.com/pixlsus/registry.gimp.org_static/raw/master/registry.gimp.org/files/wavelet-sharpen-0.1.2.tar.gz";
+        sha256 = "0vql1k67i21g5ivaa1jh56rg427m0icrkpryrhg75nscpirfxxqw";
+      };
+      installPhase = "installPlugins src/wavelet-sharpen"; # TODO translations are not copied .. How to do this on nix?
+    };
+
+    lqrPlugin = pluginDerivation {
+      # menu:
+      # Layer/Liquid Rescale
+      name = "lqr-plugin-0.6.1";
+      buildInputs = with pkgs; [ liblqr1 ];
+      src = fetchurl {
+        url = "https://github.com/pixlsus/registry.gimp.org_static/raw/master/registry.gimp.org/files/gimp-lqr-plugin-0.6.1.tar.bz2";
+        sha256 = "00hklkpcimcbpjly4rjhfipaw096cpy768g9wixglwrsyqhil7l9";
+      };
+      #postInstall = ''mkdir -p $out/nix-support; echo "${liblqr1}" > "$out/nix-support/propagated-user-env-packages"'';
```

Comment:
we probably do not need comments like this.

Reply:
Removed this and some other comments, that are probably of no use for me when maintaining glimpsePlugins.